modin/experimental/core/execution/native/implementations/hdk_on_native/dataframe/dataframe.py

```diff
@@ -2506,15 +2506,21 @@ def to_pandas(self):
         # index columns.
         if len(df.columns) != len(self.columns):
             assert self._index_cols
-            df.set_index([f"F_{col}" for col in self._index_cols], inplace=True)
-            df.index.rename(self._index_names(self._index_cols), inplace=True)
-            assert len(df.columns) == len(self.columns)
```

Comment:
this assertion statement is now missing, was the statement incorrect or it was just accidentally removed?

Reply:
It was just accidentally removed. Thanks.